delphi_esr_msgs/package.xml

```diff
@@ -1,22 +1,53 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>delphi_esr_msgs</name>
   <version>2.3.1</version>
   <description>Message definitions for the Delphi ESR</description>
+  <maintainer email="software@autonomoustuff.com">AutonomouStuff Software Team</maintainer>
   <license>MIT</license>
-
   <url type="website">http://wiki.ros.org/delphi_esr_msgs</url>
   <url type="repository">https://github.com/astuff/astuff_sensor_msgs</url>
   <url type="bugtracker">https://github.com/astuff/astuff_sensor_msgs/issues</url>
-
-  <maintainer email="software@autonomoustuff.com">AutonomouStuff Software Team</maintainer>
   <author email="dstanek@autonomoustuff.com">Daniel Stanek</author>
   <author email="jkale@autonomoustuff.com">Joe Kale</author>
```

Comment:
Should we remove Kale?

Reply:
Even though he doesn't work for AS anymore, he was still an author. Convention is to leave them to maintain credit. If you feel strongly about it I will ask up the chain. If not, better to leave it, IMO.

Reply:
Just figured the e-mail was defunct.